website/docs/jellyfish/api/icxorderbook.md

```diff
@@ -92,3 +92,43 @@ interface ICXGenericResult {
   txid: string
 }
 ```
+
+## getOrder
+
+Returns information about order or fillorder
+
+```ts title="client.icxorderbook.getOrder()"
+interface icxorderbook {
+  getOrder (orderTx: string): Promise<Record<string, ICXOrderInfo | ICXOfferInfo>>
```

Comment:
`ICXOrderInfo | ICXOfferInfo` is such a mind fuck, I always thought they were the same interface without realizing one is "Order", another is "Offer"

Reply:
yeah. the same function `getOrder()`returning both order and offer having different structures. not the best way to do it. gave some trouble when interfacing also.